### PR TITLE
New Homepage design

### PR DESCRIPTION
**Pull request by flickz**
Screenshot 2020-01-06 at 6 12 37 AM

**Current route path to view `/v2`**

#### Todo

- [ ] Add respective links, path or url to `href` attributes across the page

- [ ] For the learn more section, the desktop view needs to be completed, current look:
Screenshot 2020-01-06 at 12 14 21 AM
expected look:
Screenshot 2020-01-06 at 12 14 44 AM
I'm currently finding it difficult breaking up the items into two rows because it uses `array.map` to render. Maybe @Betree can give a suggestion but I'll get the hinge of it today

- [ ] Creation of UI components, currently have  a reusable component created from Mobile view, `StyledCarousel`, should be added to the styled guide although it is just basic state view change but uses [`react-swipeable`](https://github.com/dogfessional/react-swipeable) to detect swipe from left to right, currently lacks animation, should be added.
Screenshot 2020-01-06 at 5 51 47 AM

### CODE REVIEW

File: components/home/HomeIllustration.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import { rotateMixin } from '../../lib/constants/animations';
+
+const Illustration = styled.img`
+  max-width: 100%;
+  max-height: 100%;
+  ${space}
+  ${display}
+
+  ${({ animate }) => (animate ? rotateMixin : null)};
```
Comment (Betree) [inline components/home/HomeIllustration.js]: Is this used somewhere?

**Comment by znarf** [comment]
@flickz for you:

- [ ] In the top bar, the Open Collective logo should lead to the homepage
- [ ] Please plug all the links. Let us know if there is any blocker.
- [ ] Can you plug different screenshots, even if not final versions? So we can better see how it's reacting on change
- [ ] In "We are Open in every way", on mobile, let's make sure the height is the same in the 2 sections
- [ ] Learn more section. Not sure how we can help?

for other team members:

- [ ] Final copy for "Learn more" blocks

**Comment by xdamman** [comment]
See also here for more comments: https://github.com/opencollective/opencollective/issues/2417#issuecomment-571560926
and a link to a shared google doc for the copy.

**Comment by flickz** [comment]
@znarf I need help with the url of the following links 

"View public drive"
Screenshot 2020-01-08 at 10 44 45 AM

"Open source projects", "Meetups" "Movments"
Screenshot 2020-01-08 at 10 48 46 AM

**Comment by Betree** [comment]
Great progress! Some feedback on the visual:

- Lot of space in some sections

image

- I think these images colors should be grey until they're selected, no?

image

- Menu entries should not move when clicking on items

Peek 23-01-2020 15-05

- Button color on hover looks strange

image

**Comment by znarf** [comment]
Thank you! Merged in https://github.com/opencollective/opencollective-frontend/pull/3473

We'll follow up in smaller issues.

**Pull request closed without merging**